jjb/dynamic/SVT_Network_Performance_Test.yml

```diff
@@ -6,16 +6,16 @@
         if [[ ! -d "${WORKSPACE}/perf-dept" ]]; then
         git clone https://${GITHUB_TOKEN}@github.com/redhat-performance/perf-dept.git
         fi
-        export PRIVATE_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
+        export PRIVATE_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_pbench_ec2
         chmod 600 ${PRIVATE_KEY}
 
         printenv
 
-        #scp -i $WORKSPACE/private/config/keys/id_rsa_perf id_rsa.pub root@${OS_HOST_NAME}:/root/svt/networking/synthetic/id_rsa.pub
-        #scp -i $WORKSPACE/private/config/keys/id_rsa_perf config.yaml root@${OS_HOST_NAME}:/root/svt/networking/synthetic/config.yaml
+        #scp -i $WORKSPACE/private/config/keys/id_rsa_pbench_ec2 id_rsa.pub root@${OS_HOST_NAME}:/root/svt/networking/synthetic/id_rsa.pub
```

Comment:
Not related to this PR but the path to the keys doesn't look right for this Job but I guess we are not maintaining this job anymore. We can remove it in favor of https://github.com/openshift-scale/scale-ci-pipeline/blob/master/jjb/dynamic/uperf.yml. Let's create an issue to track it. Thoughts?

Reply:
ack, will create one